docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alectolytic/scipy
+FROM matthewfeickert/python3-scipy-alpine:scipy-1.2.1
```

Comment:
Is there a github repo somewhere to see how this is built?

Reply:
yup. Linked in the description, but also here: https://github.com/matthewfeickert/python3-scipy-alpine

Reply:
ok will we find this again if we forget? maybe link PR number as comment in the dockerfile?

Reply:
> ok will we find this again if we forget?

Sorry, I think I'm missing something simple. What is the "this" what we will forget? The Dockerfile for the base image?

Reply:
was just thinking `FROM matthewfeickert/python3-scipy-alpine:scipy-1.2.1  # issue #697` but yeah no strong opinion

Reply:
Done in https://github.com/scikit-hep/pyhf/pull/697/commits/745fed951d178e1b07b9bbf21349619a348d419a :+1:

Reply:
Oh that actually breaks things. I'm going to revert it.

Uh, unless you have other ideas maybe we just go with `git log -- docker/Dockerfile` for future reminders?

Reply:
@kratsg If you have thoughts here they'd be welcome.

Reply:
no ideas unfortunately